fission-cli/library/Fission/CLI/IPFS/Download/GitHub/Types.hs

```diff
@@ -0,0 +1,17 @@
+module Fission.CLI.IPFS.Download.GitHub.Types (GetRelease) where
+
+import qualified RIO.ByteString.Lazy            as Lazy
+
+import           Servant.API
+
+import qualified Fission.CLI.IPFS.Release.Types as IPFS
+import qualified Fission.CLI.IPFS.Version.Types as IPFS
+
+type GetRelease
+  = "ipfs"
+  :> "go-ipfs"
+  :> "releases"
+  :> "download"
+  :> Capture "version" IPFS.Version
+  :> Capture "release" IPFS.Release
+  :> Get '[OctetStream] Lazy.ByteString
```

Comment:
`Lazy.ByteString` because the tar extractor expects that, so it's more convenient. In general we default to strict because it's easier to reason about.